spec/requests/v0/profile/telephones_spec.rb

```diff
@@ -480,18 +473,16 @@
   describe 'DELETE /v0/profile/telephones v2' do
     before do
       Flipper.enable(:va_v3_contact_information_service)
-      telephone.id = id_in_cassette
     end
 
     after do
       Flipper.disable(:va_v3_contact_information_service)
     end
 
-    let(:user) { build(:user, :loa3, icn: '64762895576664260') }
```

Comment:
Not sure why a different icn was used. This was removed to use the icn defined in the :loa user factory